presto-main/src/test/java/com/facebook/presto/util/TestGraphvizPrinter.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.util;
+
+import com.facebook.presto.common.predicate.TupleDomain;
+import com.facebook.presto.cost.StatsAndCosts;
+import com.facebook.presto.metadata.FunctionAndTypeManager;
+import com.facebook.presto.operator.StageExecutionDescriptor;
+import com.facebook.presto.spi.ConnectorId;
+import com.facebook.presto.spi.ConnectorTableHandle;
+import com.facebook.presto.spi.TableHandle;
+import com.facebook.presto.spi.plan.PlanNode;
+import com.facebook.presto.spi.plan.PlanNodeId;
+import com.facebook.presto.spi.plan.TableScanNode;
+import com.facebook.presto.sql.planner.Partitioning;
+import com.facebook.presto.sql.planner.PartitioningScheme;
+import com.facebook.presto.sql.planner.PlanFragment;
+import com.facebook.presto.sql.planner.SubPlan;
+import com.facebook.presto.sql.planner.plan.PlanFragmentId;
+import com.facebook.presto.testing.TestingSession;
+import com.facebook.presto.testing.TestingTransactionHandle;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+
+import static com.facebook.presto.sql.planner.SystemPartitioningHandle.SINGLE_DISTRIBUTION;
+import static com.facebook.presto.sql.planner.SystemPartitioningHandle.SOURCE_DISTRIBUTION;
+import static org.testng.Assert.assertEquals;
+
+public class TestGraphvizPrinter
+{
+    private static final String TEST_CONNECTOR_ID_STR = "connector_id";
+    private static final ConnectorId TEST_CONNECTOR_ID = new ConnectorId(TEST_CONNECTOR_ID_STR);
+    private static final PlanNodeId TEST_TABLE_SCAN_NODE_ID = new PlanNodeId("plan_id");
+    private static final String TEST_CONNECTOR_TABLE_HANDLE_STR = "testConnectorTableHandle";
+    private static final ConnectorTableHandle TEST_CONNECTOR_TABLE_HANDLE = new ConnectorTableHandle()
+    {
+        @Override
+        public String toString()
+        {   // TestingMetadata::TestingTableHandle uses default toString, which is inconsistent
+            return TEST_CONNECTOR_TABLE_HANDLE_STR;
+        }
+    };
+
+    @Test
+    public void testPrintLogical()
+    {
+        String actualStr = GraphvizPrinter.printLogical(
+                ImmutableList.of(createTestPlanFragment(0, createTestTableScanNode())),
```

Comment:
Arguments should be on separate lines

Reply:
I thought that only applies when we need a line wrap per [here](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting)? But I never really found documentation on how long that should be, I just trusted the Airlift code style...